tests/test_size.py

```diff
@@ -56,56 +58,93 @@ def bar(self):
         else:
             pass
 """
-    check_size(example, 12)
+    check_size(example, [2, 11])
 
 
-def test_size_try_except():
+def test_size_while_else():
     example = """
-try:
-    x = sys.argv[99]
-except IndexError:
-    pass
-except Exception:
-    pass
-else:
-    pass
+class Foo:
+    while "b" > "a":
+        pass
+    else:
+        pass
 """
-    check_size(example, 7)
+    check_size(example, [5])
 
 
 def test_size_file():
     example = """
-with open("/dev/null") as f:
-    f.write("")
+class Foo:
+    with open("/dev/null") as f:
+        f.write("")
 """
-    check_size(example, 2)
+    check_size(example, [3])
 
 
-def test_size_while_else():
+def test_size_try_except_else():
     example = """
-while "b" > "a":
-    pass
-else:
-    pass
+class Foo:
+    try:
+        x = sys.argv[99]
+    except IndexError:
+        pass
+    except Exception:
+        pass
+    else:
+        pass
 """
-    check_size(example, 3)
+    check_size(example, [9])
 
 
 def test_size_try_finally():
     example = """
-try:
-    1/0
-finally:
-    return 99
+class Foo:
+    try:
+        1/0
+    finally:
+        return 99
+"""
+    check_size(example, [5])
+
+
+def test_size_try_except():
+    example = """
+class Foo:
+    try:
+        foo()
+    except:
+        bar()
 """
-    check_size(example, 3)
+    check_size(example, [5])
 
 
 def test_size_for_else():
     example = """
-for arg in sys.argv:
-    print("loop")
-else:
-    print("else")
+class Foo:
+    for arg in sys.argv:
+        print("loop")
+    else:
+        print("else")
+"""
+    check_size(example, [5])
+
+
+def test_size_class_nested():
+    example = """
+class Foo:
+    class test:
+        pass
+"""
+    check_size(example, [3])
+
+
+def test_multi_line_return():
+    example = """
+def long_string_return(o):
```

Comment:
This can become

    class Foo:
        def foo():
            return (
                'very'
                'long'
                'string')

Please add a comment: `// We currently cannot handle code ending with multiline statements.`